docs/content/querying/dimensionspecs.md

```diff
@@ -396,6 +396,10 @@ or without setting "locale" (in this case, the current value of the default loca
 
 ### Lookup DimensionSpecs
 
+<div class="note caution">
+Lookups are an experimental feature.
```

Comment:
or link to the 'experimental features' page if that already explains what experimental means.